content/en/platform/corda/4.7/enterprise/notary/installing-the-notary-service.md

```diff
@@ -180,5 +180,5 @@ dataSourceProperties = {
 
 ## Obtaining the notary service identity
```

Comment:
On this page: 
![image](https://user-images.githubusercontent.com/69467840/137348388-629b7fb5-87c2-488f-8407-e3e1f54f3492.png)
The first link to config_obfuscator is broken (maybe remove it, if it's just a link to the image?), the notary registration section reference is incorrect, and the last one should be a link.